docs/quick-start/nginx-ingress/README.md

```diff
@@ -0,0 +1,131 @@
+# Cert-Manager NGINX Ingress Quick-Start Guide
+
+## Step -1 - Install Helm Client
+**Skip this section if you have `helm` installed.**
+
+The easiest way to install `cert-manager` is to use [Helm](https://github.com/kubernetes/helm), a templating and deployment system for Kubernetes resources.
+
+Firstly, ensure you have the Helm client installed by following [its installation instructions](https://github.com/kubernetes/helm/blob/master/docs/install.md).
+
+For example, on Mac OS X:
+```bash
+$ brew install kubernetes-helm
+```
+
+
+## Step 0 - Installer Tiller
+**Skip this section if you have Tiller set-up.**
+
+Tiller is Helm's server-side component, which the `helm` client uses to deploy resources.
+
+Deploying resources is a privileged operation; in the general case requiring arbitrary privileges. As a convenience, we will give Tiller complete control of the cluster. Do not run with this configuration in production.
+
+Create the RBAC resources.
+
+```bash
+$ kubectl create serviceaccount tiller --namespace=kube-system
+$ kubectl create clusterrolebinding tiller-admin --serviceaccount=kube-system:tiller --clusterrole=cluster-admin
+```
+
+Now install the Tiller component.
+
+```bash
+$ helm init --service-account=tiller
+```
+
+*If you'd like to read more, full documentation is available on [installing Tiller](https://github.com/kubernetes/helm/blob/master/docs/install.md) and [RBAC configuration](https://github.com/kubernetes/helm/blob/master/docs/rbac.md).*
+
+
+## Step 1 - Deploy the NGINX Ingress Controller
+### Create a namespace
+```bash
+$ kubectl apply -f nginx/00-namespace.yaml
+```
+
+### Create a default backend
+Used for all requests that don't match any Ingress resource
+
+```bash
+$ kubectl apply -f nginx/default-deployment.yaml
+$ kubectl apply -f nginx/default-service.yaml
+```
+
+### Create the NGINX ingress controller
+```bash
+$ kubectl apply -f nginx/rbac.yaml
+$ kubectl apply -f nginx/configmap.yaml
+$ kubectl apply -f nginx/service.yaml
+$ kubectl apply -f nginx/deployment.yaml
+```
+
+The `nginx` ingress controller service is the only one which is made available outside the cluster, as it handles all incoming traffic. It does this via a load balancer in your cloud provider. A few minutes after you've created the `nginx` Service, get its public IP or domain name via `kubectl`.
+
+```bash
+$ kubectl describe svc nginx --namespace nginx-ingress
+[...]
+LoadBalancer Ingress:   1.2.3.4
+[...]
+```
+
+This is the IP to which all incoming traffic should be routed, so add it to a DNS zone you control, e.g. as `echo.example.com`. AWS will give a domain name instead; in this case CNAME your name to that name.
+
+
+## Step 2 - Deploy an Example Service (Echoserver)
+```bash
+$ kubectl apply -f echoserver/00-namespace.yaml
+$ kubectl apply -f echoserver/deployment.yaml
+$ kubectl apply -f echoserver/service.yaml
+```
+
+We will first create an Ingress resource that listens on plain HTTP.
```

Comment:
... so that we can ensure our ingress is properly configured and routing traffic to your echoserver service.